packages/remark-math/inline.js

```diff
@@ -2,72 +2,121 @@ function locator (value, fromIndex) {
   return value.indexOf('$', fromIndex)
 }
 
-const ESCAPED_INLINE_MATH = /^\\\$/
-const INLINE_MATH = /^\$((?:\\\$|[^$])+)\$/
-const INLINE_MATH_DOUBLE = /^\$\$((?:\\\$|[^$])+)\$\$/
+const ESCAPED_INLINE_MATH = /^\\\$/ // starts with \$
+
+const NODE_TYPES = {
+  TEXT: 'text',
+  MATH_INLINE: 'inlineMath'
+}
+
+const defaultModes = {
+  mathInlineDouble: {
```

Comment:
Oh yeah, I'm about to leave a similar comment on the other PR, but I think `inlineMathDouble` and `inlineMath` are a little more readable than this ordering, because it feels more like the regular order that you would say these words in a sentence. Also that's the order that the original regex constants used.